### ISSUE TITLE
Clean-up commented code-lines

### ISSUE THREAD

**Issue by danielpeintner** [eclipse/thingweb.node-wot#566]
A PR could handle all appearances like the following

https://github.com/eclipse/thingweb.node-wot/pull/554#discussion_r727885177

**Comment by fillobotto** [comment]
Is this still valid? As of now I can't spot any piece of commented code.
Just this one https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-opcua/test/full-opcua-thing-test.ts#L28

**Comment by relu91** [comment]
Yes, the commented code is still there. Plus I also found commented code in CoapClientTests. Since it is hard to spot commented code it would be fine to clean the blocks that we spotted here.
